DataFormats/PatCandidates/interface/PackedCandidate.h

```diff
@@ -638,6 +648,23 @@ namespace pat {
         uint16_t dphidphi;
     };
 
+    /// time (wrt nominal zero of the collision)
+    virtual float time(size_t ipv=0)  const { return vertexRef()->t() + dtimeAssociatedPV(); }
```

Comment:
what's the reason for the argument here?

Reply:
The argument is the index of the primary vertex, so one can compute the delta time with respect to any primary vertex in the collection (of slimmedOfflinePrimaryVertices).

Reply:
The reason for the question is the implementation does not depend on the input argument.

Reply:
Yes you're right, sorry, I was thinking of the dtime function just below.  I agree the argument is not needed for this one.